lib/src/components/NxLoadError/__tests__/NxLoadError.test.tsx

```diff
@@ -4,86 +4,109 @@
   });
 
-  it('renders a retry button if there is an error and retryHandler is set', function() {
-    expect(getShallowComponent({ error: 'Error!' })).not.toContainMatchingElement(NxButton);
-
-    expect(getShallowComponent({ error: 'Error!', retryHandler: () => {} })).toContainMatchingElement(NxButton);
-    expect(getShallowComponent({ error: 'Error!', retryHandler: () => {} }).find(NxButton)).toIncludeText('Retry');
+  it('renders a retry button if there is an error and retryHandler is set', async function() {
+    const elWithoutRetryButton = renderEl({ error: 'Error' });
+    expect(elWithoutRetryButton?.textContent).not.toContain('Retry');
+    expect(quickRender({ error: 'Error', retryHandler: () => {} }).getByRole('button', { name: 'Retry' }))
+        .toBeInTheDocument();
   });
 
   it('adds the appropriate class, variant, and type to the retry button', function() {
-    const button = getShallowComponent({ error: 'Error!', retryHandler: () => {} }).find(NxButton);
-
-    expect(button).toHaveClassName('nx-load-error__retry');
-    expect(button).toHaveProp('variant', 'error');
-    expect(button).toHaveProp('type', 'button');
+    renderEl({ error: 'Error', retryHandler: () => {} });
+    const retryButton = screen.getByRole('button', {name: 'Retry'});
+    expect(retryButton).toBeInTheDocument();
+    expect(retryButton).toHaveAttribute('type', 'button');
   });
 
   it('sets the retry button type to "submit" when submitOnRetry is set to true', function() {
-    const props = { error: 'Error!', submitOnRetry: true };
-    const button = getShallowComponent(props).find(NxButton);
-    expect(button).toHaveProp('type', 'submit');
+    renderEl({ error: 'Error', submitOnRetry: true });
+    const retryButton = screen.getByRole('button', {name: 'Retry'});
+    expect(retryButton).toBeInTheDocument();
+    expect(retryButton).toHaveAttribute('type', 'submit');
   });
 
-  it('calls the retryHandler when the retry button is clicked', function() {
-    const retryHandler = jest.fn(),
-        props = { error: 'Error!', canRetry: true, retryHandler },
-        component = getShallowComponent(props);
-
+  it('calls the retryHandler when the retry button is clicked', async function() {
+    const user = userEvent.setup();
+    const retryHandler = jest.fn();
+    renderEl({ error: 'Error!', retryHandler });
     expect(retryHandler).not.toHaveBeenCalled();
-
-    component.find(NxButton).simulate('click');
-
+    const retryButton = screen.getByRole('button', {name: 'Retry'});
+    expect(retryButton).toBeInTheDocument();
+    await user.click(retryButton);
     expect(retryHandler).toHaveBeenCalled();
   });
 
-  it('passes unknown props to the NxErrorAlert element', function() {
-    const onClick = jest.fn(),
-        component = getShallowComponent({ error: 'err', id: 'foo', onClick });
+  describe('when onClose is defined', ()=> {
+    it('if onClose function is defined then the "X" button is appear', function() {
```

Comment:
```suggestion
    it('the "X" button appears', function() {
```
The `describe` block already has "when onClose is defined"